docs/excel/custom-functions-debugging.md

```diff
@@ -0,0 +1,157 @@
+---
+ms.date: 02/26/2019
+description: Debug your custom functions in Excel.
+title: Custom functions debugging (preview)
+localization_priority: Normal
+---
+# Custom functions debugging (preview)
+
+Debugging for custom functions can be accomplished by multiple means, depending on what platform you're using. Methods differ between [Office Online for Windows](#windows-10-visual-studio-and-microsoft-edge), [Office Online for Mac](#mac-and-chrome-debugger), [Office Desktop for Windows](#for-windows-and-visual-studio), and [Office Desktop for Mac](#for-mac). You can also issue debugging commands through [the command line directly](#using-the-command-line).
+
+[!include[Excel custom functions note](../includes/excel-custom-functions-note.md)]
+
+> [!NOTE]
+> For simplicity, this article shows debugging in the context of using Visual Studio Code to edit, run tasks, and in some cases use the debug view. If you are using a different editor or command line tool, see the [command line instructions](#Use-the-command-line-tools-to-debug) at the end of this article.
+
```

Comment:
as a pre-condition of this article, we need a project already created, i am assuming with yo office? lets add that assumption: "if you dont have a vs code project create one here" probably is a link to the tutorial.